### ISSUE TITLE
[Question] Can I use it with "Used Space Only" encryption without protectors?

### ISSUE THREAD

**Issue by jm4R** [Aorimn/dislocker#324]
![no_protectors](https://github.com/Aorimn/dislocker/assets/38551394/efb545e1-5988-4f30-ad57-98146b7b25af)
![no_protectors2](https://github.com/Aorimn/dislocker/assets/38551394/520935bd-e1e0-459a-b968-2e4828533447)
I have disks from broken windows (recovery mode only works) and I want to backup my data. It seems that those disks doesn't have any key I may provide to dislocker. I wanted to use it without keys with following flags

```
dislocker /dev/sda1 -r -- /media/C
mount -o loop /media/bitlocker/dislocker-file /media/C
```

but it failed with a following error:

```
mount: /media/C: failed to setup loop device for /media/bitlocker/dislocker-file .
```

Is it possible to make it work?

**Comment by Justasic** [comment]
I just encountered this problem myself and had a hard time finding answers for it, dislocker said that there was no VMK or FVEK available to decrypt with even though I had what I thought was the correct recovery key. I had thought I was hosed but I decided my last ditch effort was to attempt to decrypt the volume in windows using that key but to my surprise windows had already automatically decrypted the volume without that key.

I tried to look into it a bit and found that this is likely something called a "clear key" encrypted volume, often happening when the volume is in a transitioning state by being encrypted or decrypted (turning BitLocker on or off). For whatever reason, the OEM deployment of my device had it stuck in this state so windows never treated it as a fully encrypted volume (not showing the lock icon in explorer) and I was unable to decrypt it in Linux using dislocker.

Some research into it sounds like the decryption key is stored unencrypted in the drive somewhere, which I would've assumed dislocker could find but I was wrong. I cannot find a utility (windows or linux) that can extract the key either (short of understanding the bitlocker binary format and parsing it myself). 

tl;dr: 
My solution for now is to mount the drive in another windows machine (which unlocks it automatically) then run `manage-bde -off H:` to decrypt the `H` drive. Alternatively you can set the `RecoveryPassword` key protector and that should allow dislocker to decrypt it using the recovery key (or so I would assume based on what is said online, I have not personally tested this).
